pkg/bundle/manager.go

```diff
@@ -66,6 +66,15 @@ func (m bundleManager) ProcessBundle(ctx context.Context, newBundle *api.Package
 	return false, nil
 }
 
+func (m *bundleManager) hasLatestBundle(sortedBundles []api.PackageBundle, latestBundle *api.PackageBundle) bool {
```

Comment:
This method isn't really checking if sortedBundles has the latest bundle. It's checking that sortedBundles has whichever bundle I pass in the latestBundle argument.

I'd suggest either renaming it to just `hasBundle`, or removing the `latestBundle` argument, and determining what the latest bundle is within the method itself, so as to enforce the "latest" in the name.

Also, I [opened a PR](https://github.com/TerryHowe/eks-anywhere-packages/pull/3) that tweaks some of the assumptions around sorting. I think it's better to do the sorting here if that's what's needed, rather than assuming that GetBundleList will slice (since there's no guarantee or checking that assumption will hold).